src/commcare_cloud/ansible/ansible.cfg

```diff
@@ -11,7 +11,7 @@ fact_caching_timeout = 21600
 timeout = 30
 
 inventory_plugins = ./plugins/inventory
-library = ./library
+library = ./library:~/.ansible/roles/lqueryvg.chage/library
```

Comment:
This is irritating. It works with `~/.ansible/roles` but then we hit this bug: https://github.com/ansible/ansible/issues/40561